src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -57,7 +57,98 @@ The module connects to the PS3 controller using bluetooth (or USB) and reacts ba
 | SQUARE                    | Toggles locking of the head.               |
 | R2                        | Right kick.                                |
 
-To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and run the binary. Plug the PS3 controller into the robot using a USB cable. When the controller is connected to the robot, it will be paired, and you can remove the cable.
+To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and install it on the robot. Ensure the PS3 controller is paired and connected to the right robot. Then run the `./ps3walk` on the robot to begin controlling it.
+
+#### Setup
+
+1. Install `bluez`, `bluez-utils`, and `bluez-plugins` if they have not already been installed:
+   ```
+   pacman -S --needed bluez bluez-utils bluez-plugins
+   ```
+2. Restart the system:
+   ```
+   reboot now
+   ```
+   This is only necessary if the packages were not previously installed.
+3. Start the bluetooth service:
+   ```
+   systemctl start bluetooth.service
+   ```
+
+<details>
+<summary> Start bluetooth automatically </summary>
+
+Bluetooth can be powered on automatically when the system starts. This enables the controllers to
+connect without having to run `systemctl start bluetooth.service` and `bluetoothctl power on`. This
+is not enable by default because we're currently unsure whether wifi and bluetooth are interferring
+with each other.
+
+1. Run
+   ```
+   systemctl enable bluetooth.service
+   ```
+2. Set `AutoEnable=true` in `/etc/bluetooth/main.conf`:
+   ```
+   [Policy]
+   AutoEnable=true
+   ```
+
+Now, after restarting the robot, and having previously paired the PS3 controller, the PS3 controller
+will connect to the robot automatically.
+
+<Alert type='info'>
+<code>ps3walk</code> still needs to be started in order to control the robot.
```

Comment:
```suggestion
`ps3walk` still needs to be started in order to control the robot.
```

Reply:
I don't think it works inside of an `<Alert>` because it changes into a HTML context.

Reply:
Should work https://nubook.nubots.net/kitchen-sink/alerts